repl/repl_test.go

```diff
@@ -1777,6 +1777,79 @@ Redo data.a[i].b.c[j] = x; data.a[k].b.c[x] = 1
 	}
 }
 
+func TestEvalTraceIndex(t *testing.T) {
```

Comment:
It's not clear to me that this test case is necessary. The REPL doesn't have any special logic to deal with index trace events. However, if we change anything about the traces (e.g., the output format) then this test case is going to need to be updated. When you update the other test case, could you remove this one?

Reply:
Removed